### PR TITLE
gnome-extra/nm-applet: fix bashisms in autoconf

### PR DESCRIPTION
**Pull request by EvaSDK**
Package-Manager: Portage-2.3.99, Repoman-2.3.22

dash is my /bin/sh and it made this package block upgrades on my desktop.

### CODE REVIEW

**Comment by EvaSDK** [comment]
NetworkManager issue filled upstream as https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/472

**Comment by anarchpenguin** [comment]
ack

**Comment by EvaSDK** [comment]
Added missing sign off

**Comment by leio** [comment]
Did the networkmanager patch get updated with the changes done during upstream review at https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/472 ?

File: dev-libs/libreport/libreport-2.12.0.ebuild [hunk 1]
```diff
@@ -50,6 +50,10 @@ BDEPEND="
 # Tests require python-meh, which is highly redhat-specific.
 RESTRICT="test"
 
+PATCHES=(
+	"${FILESDIR}"/${PV}-fix-bashisms.patch
```
Comment (thesamesam) [inline dev-libs/libreport/libreport-2.12.0.ebuild:54]: Usually ${PN} is appended?

**Comment by leio** [comment]
I'm pretty sure we are good with the networkmanager stack on this by now, as the upstreamed stuff trickled down. Not sure about libreport though, which is one of the commits in here

**Pull request closed without merging**